css/index.css

```diff
@@ -1,3 +1,362 @@
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+	display: flex;
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+	display: flex;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+.main-services-section {
+}
+
+.top-col {
+	display: flex;
+	vertical-align: middle;
+}
+
+.middle-col {
+	display: flex;
+}
+
+.last-col {
+	display: flex;
+}
+
+.our-services-col {
+	display: flex;
+}
+
+
+
+.item1 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item1 p {
+	padding-bottom: 20px;
+}
+
+.item2 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item2 p {
+	padding-bottom: 20px;
+}
+
+.item3 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item3 p {
+	padding-bottom: 20px;
+}
+
+.item4 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item4 p {
+	padding-bottom: 20px;
+}
+
+.item5 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item5 p {
+	padding-bottom: 20px;
+}
+
+.item6 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item6 p {
+	padding-bottom: 20px;
+}
+
+.item7 {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+}
+
+.section-box {
+	height: 225px;
+}
+
+/* tablet */
+
+@media (max-width: 800px) {
+  header {
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+  }
+
+  header nav {
+    display: flex;
+    width: 691px;
+  }
+
+  header nav a {
+    display: flex;
+    justify-content: center;
+  }
+
+  .cta-text {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .cta img {
+    display: none;
+  }
+}
+
+/* smartphone */
+
+@media (max-width: 500px) {
+  header nav {
```

Comment:
if the nav already had a flex property on it you don't need to specify it again it will inherit it